python/GafferSceneUI/CameraUI.py

```diff
@@ -46,59 +49,237 @@
 import GafferScene
 import GafferSceneUI
 
-filmFitMetadata = [
+renderOptionsMetadata = {
 
-	"description",
-	"""
-	Determines how the size of the rendered image relates to the camera aperture. If the aspect
-	ratios of the aperture and the output resolution are the same, then this has no effect,
-	otherwise it dictates what method is used to preserve the pixel aspect ratio of the rendered
-	image.
-
-	Horizontal
-	:   The frustum is adjusted so that the rendered image fills the full
-		width of the aperture and aspect ratio is preserved.
-
-	Vertical
-	:   The frustum is adjusted so that the rendered image fills the full
-		height of the aperture and aspect ratio is preserved.
-
-	Fit
-	:   Automatically picks Horizontal or Vertical such that all of the aperture is contained
-		within the output image. This may result in seeing outside the aperture at the top and
-		bottom or left and right.
-
-	Fill
-	:   Automatically picks Horizontal or Vertical such that the output image is fully covered by
-		the aperture. Part of the aperture may be cropped off at the top and bottom or left and
-		right.
-
-	Distort
-	:   Distorts the frustum so that the aperture is fitted exactly to the output display window,
-		resulting in non-square pixels
-	""",
+	"filmFit" : [
+
+		"description",
+
+		"""
+		How to fit the aperture within the image of the render. Fitting
+		is applied only if the `width / height` aspect ratios of the
+		aperture and the image are different. The following fitting
+		modes are available:
+
+		<ul>
```

Comment:
We lose some of the source readability here, but this list markup works in both the app and the Reference docs, which are a higher priority for me.

Reply:
Ugh. I have a branch where I do the markdown->html conversion on the fly for tooltips. Let me see if I can dust that off so we can keep the source readable.

Reply:
I've made a PR for markdown-in-tooltips here : https://github.com/GafferHQ/gaffer/pull/3065. If you could take a look that would be great...